umm-spec-lib/src/cmr/umm_spec/models/umm_variable_models.clj

```diff
@@ -5,31 +5,34 @@
 
 (defrecord UMM-Var
   [
-   ;; Valid range data value of a variable: minimum and maximum values. For example. '7500, -1', or
-   ;; '0, 377'.
-   ValidRange
-
-   ;; The service information of a variable.
-   Service
+   ;; Valid ranges of variable data values.
+   ValidRanges
 
+   ;; A variable consists of one or more dimensions. An example of a dimension name is 'XDim'. An
+   ;; example of a dimension size is '1200'. Variables are rarely one dimensional.
    Dimensions
 
-   ;; Scale factor which has been applied to the cell value. For example, '0.002'
+   ;; The factor by which the cell value has been multiplied. The cell value is the value given to a
+   ;; specific location within an array of data values. An example of a scale factor is '0.002'
    Scale
 
-   ;; The physical value corresponding to a cell value of zero. For example, '0.49'.
+   ;; The factor to which the cell value has been added. The celle value is the value given to a
```

Comment:
This is from the other schema updates that are not part of this issue. I will update the description in UMM repo and it will show up in CMR next time we update the schema.